Neos.ContentRepository.Core/Classes/NodeType/NodeType.php

```diff
@@ -59,7 +54,7 @@ class NodeType
     /**
      * Is this node type marked abstract
      */
-    public bool $abstract = false;
+    protected bool $abstract = false;
```

Comment:
Why `protected` and not `private`?

Reply:
this was a regression see https://github.com/neos/neos-development-collection/pull/4520/commits/5acb733e73a79f917e9e2dc713ee31d554a81996 and was just 1 to 1 reverted. I wont attempt to cleanup the nodetype in this regard in this pr ;)